docker/build-tools/Dockerfile

```diff
@@ -324,10 +326,10 @@ ENV LANG=C.UTF-8
 ENV GO111MODULE=on
 ENV GOPROXY=https://proxy.golang.org
 ENV GOSUMDB=https://sum.golang.org
-ENV GOPATH=/go
+ENV GOPATH=/work/out:/gotools
 ENV GOROOT=/go
 ENV GOCACHE=/gocache
-ENV PATH=/go/bin:$PATH
+ENV PATH=/go/bin:/gotools/bin:$PATH
```

Comment:
How about we put all the gotools/bin into /usr/bin instead of a special directory? They're just binaries like any other, no sense in segregating them.

Reply:
not sure if that will work - gotools ends up with src and pkg files as well.  Will give it a spin.

Reply:
That's the thing, for the tools we build, we don't need any of the /src or /pkg stuff. We just need the final binaries. So just copy the bin directory contents over.

Reply:
PTAL.  This should be addressed in the latest update (after I tested this manually against the website repository and api repository).  API repository needs a rev as cue was renamed and the protos don't build properly with the latest image.  I'll roll out the latest image to api repo and make the necessary changes to that repository using this image.  (IOW need this to merge to fix up API).

Reply:
note istio repository was tested as well with this image.  It needs one minor change (to TOP_GO environment) and continues to not work because of lack of a running docker daemon.

Will need this new image - and roll out to the following repos -> api, istio, website, (and possibly operator).
Cheers
-steve